client/src/components/dashboard/Preferences.js

```diff
@@ -17,14 +17,14 @@ import propTypes from 'prop-types';
 import React from 'react';
 import { savePreferencesViewState } from '../../actions/views';
 import { saveUser, updateUser, updateUserPartial } from '../../actions/user';
-import skills from '../../assets/dropdowns/skills';
+import skills, { SKILLS_MAP } from '../../assets/dropdowns/skills';
 import SkillsAndInterests from './Profile/Preferences/SkillsAndInterests';
 import Social from './Profile/Preferences/Social';
 import styled from 'styled-components';
 import swearjar from '../../assets/helpers/swearjar-lite';
 import { ThickPaddedBottom } from '../../styles/style-utils';
 import UserLabel from '../dashboard/common/UserLabel';
-import Validator from 'validator';
+import __ from '../../assets/helpers/validations';
```

Comment:
@bonham000 I just went with the first thing before I read this... does that make it more clear like you are saying?